lms/djangoapps/dashboard/git_import.py

```diff
@@ -59,6 +59,7 @@ def cmd_log(cmd, cwd):
     command doesn't return 0, and returns the command's output.
     """
     output = subprocess.check_output(cmd, cwd=cwd, stderr=subprocess.STDOUT)
+    # pylint: disable=logging-format-interpolation
```

Comment:
I could put this back in; my issue here was that I thought you can't do the `{0!r}` formatting with `%` interpolation. But http://pyformat.info/#conversion_flags suggests I'm wrong, so I'll use `%r` and remove this disable.